style/ruby/.rubocop.yml

```diff
@@ -100,7 +100,7 @@ Rails/Delegate:
   Description: 'Prefer delegate method for delegations.'
   Enabled: false
 
-Style/DeprecatedHashMethods:
+Style/PreferredHashMethods:
   Description: 'Checks for use of deprecated Hash methods.'
```

Comment:
Whoops, missed that. I only noticed it because of the warning raised. Happy to change that as well, if wanted.